assets/_scss/core/_utilities.scss

```diff
@@ -88,7 +88,7 @@
     @include media($medium-screen) {
       float: left;
       margin-right: 2em;
-      margin-top: .5em;
+      margin-top: 0.5em;
```

Comment:
This PR was mostly about consistent formatting and didn't look at `em` vs `rem`.